kedro/templates/project/hooks/post_gen_project.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+from kedro.templates.project.hooks.utils import (
+    parse_add_ons_input,
+    setup_template_add_ons,
+    sort_requirements,
+)
+
+def main():
+    current_dir = Path.cwd()
+    requirements_file_path = current_dir / "src/requirements.txt"
+    pyproject_file_path = current_dir / "pyproject.toml"
+    pyproject_src_file_path = current_dir / "src/pyproject.toml"
```

Comment:
@astrojuanlu Reminder that we should fix https://github.com/kedro-org/kedro/issues/2975 asap, so don't have to go back and change too many things.

Reply:
Yep, better to get that ASAP. May I rebase https://github.com/kedro-org/kedro/pull/2926 on top of `develop` and open it for reviews at this point?

Reply:
Yes go for it @astrojuanlu 💪